packages/@tinacms/graphql/src/primitives/schema/index.ts

```diff
@@ -108,7 +108,7 @@ export class TinaSchema {
   }> => {
     let template
     const collection = this.getCollections().find((collection) => {
-      return filepath.startsWith(collection.path)
+      return filepath.replace('\\','/').startsWith(collection.path)
```

Comment:
Should we also use `normalize-path` here, as well?  Set `filepath.startsWith(collection.path)` to some variable and run `normalize-paths` against it?  Or is this solving _another_ Windows-related problem?

Reply:
I'm not sure we need that, it's comparing `my/path` as a string